jenkins/bootstrap_test.py

```diff
@@ -982,7 +984,12 @@ def testOnlyJobs(self):
             self.assertTrue(os.access(job_path, os.X_OK|os.R_OK), job_path)
 
     def testAllProjectAreUnique(self):
-        projects = collections.defaultdict(list)
+        allowed_list = {  # TODO(fejta): remove these
```

Comment:
sgtm, then maybe add some comments for this TODO since I would not understand by reading the code..